### PR TITLE
Doc: Add System Links

### PR DESCRIPTION
**Pull request by ax3l**
Add more information on system user guides, and production directories so people have a quick time starting with PIConGPU.

### CODE REVIEW

File: docs/source/install/profile.rst [hunk 1]
```diff
@@ -38,6 +44,12 @@ Queue: gpu (4x NVIDIA P100 16GB)
 Hypnos (HZDR)
 -------------
 
+**System overview:** `link (internal) <https://www.hzdr.de/db/Cms?pOid=29813>`_
+
+**User guide:** *None*
```
Comment (ax3l) [inline docs/source/install/profile.rst]: Isn't there a funny web server mini-wiki running on `http://hypnos4/wiki` or so?

Comment (PrometheusPi) [reply docs/source/install/profile.rst]: I do not know - that is the first time I hear this.

Comment (psychocoderHPC) [reply docs/source/install/profile.rst]: it is `http://hypnos3/wiki` but only available within our intranet.

File: docs/source/install/profile.rst [hunk 2]
```diff
@@ -38,6 +44,12 @@ Queue: gpu (4x NVIDIA P100 16GB)
...
+
+**Production directory:** ``/bigdata/hplsim/`` with ``external/``, ``scratch/``, ``development/`` or ``production/``
```
[the remark this answers is not in the document]
Comment (ax3l) [reply docs/source/install/profile.rst]: had that initially as well. now changed back.

File: docs/source/install/profile.rst [hunk 3]
```diff
@@ -103,6 +134,12 @@ For this profile to work, you need to download the :ref:`PIConGPU source code <i
 Taurus (TU Dresden)
 -------------------
 
+**System overview:** `link <https://tu-dresden.de/zih/hochleistungsrechnen/hpc>`_
+
+**User guide:** `link <https://doc.zih.tu-dresden.de/hpc-wiki/bin/view/Compendium/SystemTaurus>`_
+
+**Production directory:** ``/scratch/$USER/`` and ``/scratch/$proj/``
```
Comment (PrometheusPi) [inline docs/source/install/profile.rst]: `$proj` is not defined by default

Comment (PrometheusPi) [reply docs/source/install/profile.rst]: you could write:
```bash
/scratch/`groups | awk '{print $1}'`/
```

Comment (ax3l) [reply docs/source/install/profile.rst]: nah, we define `$proj` in the Titan profiles already and should add it to the taurus `picongpu.profile` as well.

Comment (PrometheusPi) [reply docs/source/install/profile.rst]: I am fine with that.

Comment (ax3l) [reply docs/source/install/profile.rst]: ok, will do as a follow-up

**Pull request merged**